test/scripts/install-provisioner-working.sh

```diff
@@ -0,0 +1,64 @@
+#!/usr/bin/env bash
```

Comment:
Please run `chmod +x test/scripts/install-provisioner-working.sh` and commit the result.